ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift

```diff
@@ -155,6 +165,25 @@ class NavigationControllerDemoController: DemoController {
         return controller
     }
 
+    private func changeStyleContinuously(in navigationItem: UINavigationItem) {
```

Comment:
This sort of pattern is definitely convenient, but it really makes automated tests quite difficult